.test-infra/jenkins/job_PostCommit_Java_Nexmark_Direct.groovy

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import common_job_properties
+import nexmark_bigquery_properties
+
+// This job runs the suite of ValidatesRunner tests against the Direct runner.
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_Nexmark_Direct', 'Run Direct Nexmark',
```

Comment:
To turn off the phrase triggering let's add NexmarkPostCommitBuilder that extends the PostCommitBuilder to explicitly not use the phrase triggering. It seems a low effort and it will be more explicit than setting the trigger phrase to an empty string or null. It will also make things less coupled. We can allways swich to PostCommitBuilder if we need the exact functionality (when the phrase triggering will be there).

Reply:
+1 to the target (extending) but I would  call it NoPhraseTriggeringPostCommitBuilder because this concept is not really related to Nexmark.